includes/functions/functions_dates.php

```diff
@@ -262,13 +274,19 @@ function zen_checkdate($date_to_check, $format_string, &$date_array)
  */
 function zen_is_leap_year($year)
 {
-    if ($year % 100 == 0) {
-        if ($year % 400 == 0) return true;
-    } else {
-        if (($year % 4) == 0) return true;
+    if ($year % 4 != 0) {
```

Comment:
If not divisible by 4, then definitely not a leap year. 
This check is correct.